tests/unit/helpers.test.ts

```diff
@@ -0,0 +1,37 @@
+import { getEtaDate } from '../../lib/helpers';
+
+describe('getEtaDate', () => {
+  it('should return a date string for YYYY-MM-DD', () => {
+    // dates
+    expect(getEtaDate('eta:2022-11-14')).toBe('2022-11-14');
+    expect(getEtaDate('eta : 2022-11-14')).toBe('2022-11-14');
+    expect(getEtaDate('eta: 2022-11-14')).toBe('2022-11-14');
+  });
+
+  it('should return a date string for quarters and optional years', () => {
+    // quarters
+    expect(getEtaDate('eta: q4 2022')).toBe('2022-12-31');
+    expect(getEtaDate('eta: q3 2024')).toBe('2024-09-30');
+    expect(getEtaDate('eta: q2')).toBe(`${new Date().getFullYear()}-06-30`);
+  });
+
+  it('should return a date string for months and optional years', () => {
+    // months + years
+    expect(getEtaDate('eta: dec 2022')).toBe('2022-12-31');
+    expect(getEtaDate('eta: july 2024')).toBe('2024-07-31');
+    expect(getEtaDate('eta: jan')).toBe(`${new Date().getFullYear()}-01-31`);
+  });
+
+  it('should try to parse random dates', () => {
+    // random dates
+    expect(getEtaDate('eta: 9 dec 2022')).toBe('2022-12-09');
+    expect(getEtaDate('eta: july 4, 2024')).toBe('2024-07-04');
+    expect(getEtaDate('eta: jan 25')).toBe(`${new Date().getFullYear()}-01-31`);
```

Comment:
we shouldn't be flexible with inputs. Strict types and inputs to functions reduce complexity and increase stability. The less strict we are with date inputs, the more strange inputs will become overtime, with others expecting us to maintain the human-wide gamut of individual intuition regarding dates. 

ISO8601 (yr 2000 update) solidifies `YYYY-MM-DD` as a standard format and one we should enforce our roadmap creators to align with. See https://en.wikipedia.org/wiki/ISO_8601#Calendar_dates

Reply:
done.